packages/happn-cluster/lib/services/replicator.js

```diff
@@ -59,8 +59,8 @@ Replicator.prototype.stop = function (options, callback) {
   callback();
 };
 
-Replicator.prototype.start = require('util').promisify(function (callback) {
-  property(this, 'localClient', this.happn.services.orchestrator.members.__self.client);
+Replicator.prototype.start = util.promisify(function (callback) {
```

Comment:
Why do we need all this promisify stuff here? We going up a major - may as well clean things up with async / await

Reply:
done